integration-tests/cucumber-tests-platform-smartmetering/src/test/resources/features/osgp-adapter-ws-smartmetering/installation/SetSubscriptionInformation.feature

```diff
@@ -0,0 +1,20 @@
+@SmartMetering @Platform
+Feature: SmartMetering Installation - Set Subscription Information
+  As a grid operator
+  I want to be able to update the subscription information of a smart meter
+
+  Scenario: Set Subscription Information
+    Given a smart meter
+      | DeviceIdentification | TEST1024000000001 |
+      | DeviceType           | SMART_METER_E     |
+    When receiving a smartmetering set subscription information request
+      | DeviceIdentification | TEST1024000000001 |
+      | IpAddress            | 10.0.0.1          |
+      | BtsId                | 20                |
+      | CellId               | 1                 |
+    Then the set subscription information response should be returned
+      | DeviceIdentification | TEST1024000000001 |
+      | Result               | OK                |
+      | IpAddress            | 10.0.0.1          |
+      | BtsId                | 20                |
+      | CellId               | 1                 |
```

Comment:
Numbers should be aligned to the right

Reply:
corrected, but actually, the formatter puts them on the left side.